contrib/http_util/README.md

```diff
@@ -0,0 +1,35 @@
+# OpenCensus HTTP Util
+[![Build Status][travis-image]][travis-url]
+[![Windows Build Status][appveyor-image]][appveyor-url]
+[![Maven Central][maven-image]][maven-url]
+
+The *OpenCensus HTTP Util for Java* is a collection of utilities for trace instrumentation when
+working with HTTP.
+
+## Quickstart
+
+### Add the dependencies to your project
+
+For Maven add to your `pom.xml`:
+```xml
+<dependencies>
+  <dependency>
```

Comment:
To be correct you need a dependency on the opencensus-api as well

Reply:
Done.